iot.go

```diff
@@ -115,19 +115,19 @@ func GetRegistryCredentials(registry string, region string, s *Service) *Registr
 	req.Header.Add("ClearBlade-UserToken", s.ServiceAccountCredentials.Token)
 	resp, err := s.client.Do(req)
 	if err != nil {
-		log.Fatalln(err)
+		return nil, err;
```

Comment:
Might want to run go fmt to get rid of this semicolon. Or just remove it manually